packages/database/src/migrations/20210205033535-AddSTRIVEMolecularDataOverlays-modifies-data.js

```diff
@@ -0,0 +1,176 @@
+'use strict';
+
+import { insertObject } from '../utilities';
+import { generateId } from '../utilities/generateId';
+
+var dbm;
+var type;
+var seed;
+
+/**
+  * We receive the dbmigrate dependency from dbmigrate initially.
+  * This enables us to not have to rely on NODE_PATH.
+  */
+exports.setup = function(options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const BASE_OVERLAY = {
+  userGroup: 'STRIVE User',
+  dataElementCode: 'value',
+  isDataRegional: true,
+  measureBuilder: 'countEventsPerOrgUnit',
+  presentationOptions: {
+    values: [
+      {
+        value: 'other',
+        color: 'blue',
+      },
+      {
+        value: null,
+        color: 'grey',
+      },
+    ],
+    displayType: 'radius',
+    measureLevel: 'Facility',
+    hideFromLegend: true,
+  },
+  countryCodes: '{PG}',
+  projectCodes: '{strive,explore}',
+};
+
+const STR_QMAL05_OVERLAY = {
```

Comment:
a) I like the way that you've reduced code duplication with BASE_OVERLAY

b) I know it's nitpicky, so feel free to ignore me here, but there's still a bit of duplication that could be removed here. A pattern that could work here (or next time) is 
```javascript
OVERLAYS_TO_IMPORT = [
   {
      id: 'something',
      dataElementCode: 'abc',
      name: 'boo!',
      programCode: 'hi',
   },
   //...
]
```
And then in the up migration:
```javascript
OVERLAYS_TO_INSERT.forEach(
({ id, name, dataElementCode, programCode }) => {
    overlay = {
      ...BASE_OVERLAY,
      id,
      name,
      measureBuilderConfig: {
        dataValues: {
          [dataElementCode]: 1,
        },
        programCode,
        entityAggregation: {
          aggregationType: 'REPLACE_ORG_UNIT_WITH_ORG_GROUP',
          dataSourceEntityType: 'village',
          aggregationEntityType: 'facility',
        },
      };
    // Import etc
});
```

Just a thought